resources/test-password-logging.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
```

Comment:
what about "set -e" as a failsafe so we dont miss an error again? Afaik it should also stop the script when anything errors out unexpectedly.